grammar.js

```diff
@@ -256,7 +250,7 @@ module.exports = grammar({
     objectMethod: $ => seq($.methodHeader, "=", $._expr),
 
     objectEntry: $ => seq(
-      alias($._open_entry_bracket, "["),
+      "[",
```

Comment:
We no longer need a scanner for open entry brackets, because they are sufficiently differentiated from subscripts.